Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,38 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the
+    /// by setting URLSession.sslCertificateAuthorityFile to a file URL.
+    /// Downloadable here: https://curl.haxx.se/docs/caextract.html
+    /// Alternatively you can bypass SSL peer verification altogether:
+    /// URLSession.sslUnsafeBypassPeerVerify = true (not advised)
+    fileprivate static var _sslCertificateAuthorityFile: UnsafeMutablePointer<Int8>?
```

Comment:
Shouldn't this be `_sslCertificateAuthorityFilePathPtr` or something like that? It's not simply the backing variable for `sslCertificateAuthorityFile`.